client/.travis.yml

```diff
@@ -0,0 +1,23 @@
+---
+language: node_js
+node_js:
+  - "0.12"
```

Comment:
0.12? Wouldn't we want the latest stable which is like 5.4? Or the LTS 4.x series?